README.md

```diff
@@ -34,29 +37,40 @@ Before you begin, you need to install the following tools:
 
 To get started with Scaffold-ETH 2, follow the steps below:
 
-1. Clone this repo & install dependencies
+1. Install from NPM Registry and follow the CLI instructions.
 
 ```
-git clone https://github.com/scaffold-eth/scaffold-eth-2.git
-cd scaffold-eth-2
-yarn install
+npx create-eth@latest
 ```
 
+> 💬 Hint: If you choose Foundry as solidity framework in the CLI, you must have Foundryup previously installed in your machine. Checkout: [Getfoundry.sh](https://getfoundry.sh)
```

Comment:
> you must have Foundryup previously installed in your machine

I'd replace this with something like "you'll also need Foundryup installed in your machine". The current wording makes me think Foundryup is a requirement needed _before_ using the cli, but it can actually be installed after using the cli

Reply:
Thx for the review, I've just pushed the update on that explanation 🙌